src/vcpkg/commands.update-baseline.cpp

```diff
@@ -0,0 +1,192 @@
+#include <vcpkg/base/files.h>
+#include <vcpkg/base/messages.h>
+
+#include <vcpkg/commands.update-baseline.h>
+#include <vcpkg/configuration.h>
+#include <vcpkg/vcpkgcmdarguments.h>
+#include <vcpkg/vcpkgpaths.h>
+
+namespace
+{
+    namespace msg = vcpkg::msg;
+    DECLARE_AND_REGISTER_MESSAGE(UpdateBaselineNoConfiguration,
+                                 (),
+                                 "",
+                                 "there is no `vcpkg-configuration.json` nor manifest file to update.");
```

Comment:
```suggestion
                                 "neither `vcpkg.json` nor `vcpkg-configuration.json` exist to update.");
```

I think we've been referring to "the manifest" as "vcpkg.json + vcpkg-configuration.json" ? (And also sometimes control files...)

I think just naming the thing will be clearer for folks.